en/guide/general_usage.md

```diff
@@ -0,0 +1,59 @@
+
+The various classes also all provide methods getting human readable strings from their associated enum (e.g. [DroneCore::connection_result_str()](h../api_reference/classdronecore_1_1_drone_core.md#classdronecore_1_1_drone_core_1a84c40dcefcafe888c38a5ed8dd93b0af), [Telemetry::result_str()](../api_reference/classdronecore_1_1_telemetry.md#classdronecore_1_1_telemetry_1a05c6355b7f8743250b2a7a611ea5fb4a)). You can see how these are used in the example code.
+
+## API Limitations/Behaviour
+### Supported Vehicles
+
+DroneCore has been designed to manage *aircraft* (VTOL, copter, fixed wing) that use the PX4 autopilot. 
+
+The APIs include methods that do not make sense for other vehicle types - including "takeoff" and "land". While ground vehicles may work, they are not supported and are untested.
+
+
+### Telemetry/Information
+
+DroneCore `Telemetry` information may not precisely reflect the state of the connected vehicle.
+
+DroneCore gets and stores vehicle state information from received MAVLink messages. The API is likely to more accurately reflect the device state if DroneCore is queried just after receiving an update rather than just before getting the next one.
+
+The main implication is that developers need to use a channel and an update rate that allows them to perform the desired control algorithm - there is no point trying to use computer vision over an unreliable high-latency link. The rate at which update messages should be sent can be specified by using API (but will be limited by the bandwidth of the channel). 
+
+`Info` information is static for a particular vehicle, so should remain accurate.
+
+
+### Actions/Offboard
+
+Actions methods (and any other "vehicle instructions") return when the vehicle has confirmed that the message was received and will be acted on (or not). The methods do not wait for the commanded action to complete.
+
+So, for example, the [Action::land()](../api_reference/classdronecore_1_1_action.md#classdronecore_1_1_action_1ad1a50dd7bff99d3099916576efbf8cf6) method returns as soon as the vehicle confirms it will land, but will actually land at some later point. 
+
+The implication is that developers will need to separately monitor the completion of the requested actions, if this is important to the application.
+
+
+### Missions
+
+The `Mission` and `MissionItem` APIs provide a the most useful *subset* of MAVLink mission commands as a developer-friendly API. 
+
+Not every mission command behaviour supported by the protocol and PX4 will be supported by DroneCore. For example, at time of writing the API does not allow you to specify commands that jump back to previous commands.
+
+### Connection Status
+
+Connection state information may be up to 3 seconds out of date (callbacks are notified 3 seconds after the last heartbeat message).

```

Comment:
Well, the connection state is still ok. It is only considered "timed out" after 3 seconds... 😄

Reply:
Yoiks. Fixed.